### PR TITLE
Azure AD Identity Protection Rules

### PR DESCRIPTION
**Pull request by MarkMorow**
Summary of the Pull Request
Adds the majority of azure identity protection rules.

Detailed Description of the Pull Request / Additional Comments We have a few additional event types that still need to be added after doing some internal validation.

Example Log Event
See https://learn.microsoft.com/en-us/graph/api/resources/riskdetection?view=graph-rest-1.0#riskeventtype-values for specific events

Fixed Issues
Spelling fix on anomalous_token.yml from protectection to protection.

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/cloud/azure/identity_protection/azure_identity_protection_malware_linked_ip.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+title: Malware Linked IP
```
Comment (nasbench) [inline rules/cloud/azure/identity_protection/azure_identity_protection_malware_linked_ip.yml:1]:
```suggestion
title: Sing-In From Malware Infected IP
```

File: rules/cloud/azure/identity_protection/azure_identity_protection_new_coutry_region.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+title: New Country
+id: adf9f4d2-559e-4f5c-95be-c28dff0b1476
+status: experimental
+description: Detection considers past activity locations to determine new and infrequent locations.
```
Comment (nasbench) [inline rules/cloud/azure/identity_protection/azure_identity_protection_new_coutry_region.yml:4]:
```suggestion
description: Detects sign-ins from new countries. The detection considers past activity locations to determine new and infrequent locations.
```

File: rules/cloud/azure/identity_protection/azure_identity_protection_password_spray.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+title: Password Spray
```
Comment (nasbench) [inline rules/cloud/azure/identity_protection/azure_identity_protection_password_spray.yml:1]:
```suggestion
title: Password Spray Activity
```

File: rules/cloud/azure/identity_protection/azure_identity_protection_suspicious_browser.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+title: Suspicious Browser
```
Comment (nasbench) [inline rules/cloud/azure/identity_protection/azure_identity_protection_suspicious_browser.yml:1]:
```suggestion
title: Suspicious Browser Activity
```

**Review by nasbench — APPROVED** [review]

File: rules/cloud/azure/identity_protection/azure_identity_protection_leaked_credentials.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
+title: Leaked Credential
```
[the remark this answers is not in the document]
Comment (MarkMorow) [reply rules/cloud/azure/identity_protection/azure_identity_protection_leaked_credentials.yml:1]: Should be Azure AD account, not AD.

[the remark this answers is not in the document]
Comment (nasbench) [reply rules/cloud/azure/identity_protection/azure_identity_protection_leaked_credentials.yml:1]: Didn't know if you wanna use entra or azure. Changed to Azure.

**Comment by nasbench** [comment]
@MarkMorow Didn't wanna commit the suggestions. Left that to you in case you have feedback.

File: rules/cloud/azure/identity_protection/azure_identity_protection_malware_linked_ip.yml (hunk 1 above)
Comment (MarkMorow) [reply rules/cloud/azure/identity_protection/azure_identity_protection_malware_linked_ip.yml:1]: "Sign-in" not "sing"

File: rules/cloud/azure/identity_protection/azure_identity_protection_leaked_credentials.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (MarkMorow) [reply rules/cloud/azure/identity_protection/azure_identity_protection_leaked_credentials.yml:1]: Everywhere in this repo we should do a bulk rename of Azure AD to Entra ID but I kept it consistent with everything else.

**Comment by MarkMorow** [comment]
@nasbench  I tried to name them as the same names they are in the Identity Protection console for ease of use. If you think these make more sense for this community that's fine with me.

**Comment by nasbench** [comment]
I noticed that. I only changed the ones that were too generic to link to Azure. Since in a detection pipeline where the title is one of the first things you see. Having an alert such as "suspicious browser" for example isn't indicative of much.

Other than that everything else LGTM.

Thanks for the contribution 🙏

**Review by phantinuss — APPROVED** [review]

**Pull request merged**